indico/htdocs/sass/modules/_reviewing.scss

```diff
@@ -492,3 +492,9 @@
         }
     }
 }
+
+.reviewing-item-content {
```

Comment:
Not consistent with the other classes. I would rename this to `.review-item-content`.